### ISSUE TITLE
location of jars

### ISSUE THREAD

**Issue by RobinL** [moj-analytical-services/splink_graph#69]
I think the jars may go in `python3.6/site-packages/jars` rather than `python3.6/site-packages/splink_graph/jars`.

Need to experiment with pyproject.toml and the location of /jars in this repo.

**Comment by mamonu** [comment]
yes. i had found that out when trying to get cc working on DAP. will need to experiment with the toml file for sure.

**Comment by RobinL** [comment]
Yep - only found out because just was on the ONS Splink working group call and noticed it in one of the paths :-). not urgent

**Comment by RobinL** [comment]
It looks like anything in the `./splink_graph` folder (i.e. the folder containing `__init__.py`) will get bundled and installed so I think the solution is just to put `jars` in that folder
